### ISSUE TITLE
Package API Documentation

### ISSUE THREAD

**Issue by mariopro** [thujohn/twitter#237]
Hi, this looks like a great extension but, unless I'm not finding it in the right place, it's hard to know which parameters should be passed to each method, unless we dig into the code and find it. So the question is, is there any place where we can find the minimum required docs?
Thanks!

**Comment by atymic** [comment]
The best place to look is the actual twitter documentation :)

Though we should improve ours (auto generating API docs would be a good idea).

**Comment by tcgumus** [comment]
i agree this would be very useful. it is a great package by the way.
